server/flags-config/package.json

```diff
@@ -10,7 +10,8 @@
     "dev:push": "INSTANT_CLI_DEV=1 instant-cli push"
   },
   "dependencies": {
-    "@instantdb/core": "^0.17.7",
-    "instant-cli": "^0.17.7"
-  }
+    "@instantdb/core": "^0.17.12",
+    "instant-cli": "^0.17.12"
+  },
+  "packageManager": "pnpm@8.15.6+sha512.77b89e9be77a2b06ad8f403a19cae5e22976f61023f98ad323d5c30194958ebc02ee0a6ae5d13ee454f6134e4e8caf29a05f0b1a0e1d2b17bca6b6a1f1159f86"
```

Comment:
qq: is this intended?

Reply:
It's what pnpm did when I did pnpm i, so I assume it's kosher?

Reply:
🫡 sgtm, let's roll

Reply:
Looks like it's a corepack thing. This fella doesn't like it, but I guess we can disable it later if there's a problem https://blog.hyperknot.com/p/corepacks-packagemanager-field